includes/discovery/mempools/ironware-dyn.inc.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if ($device['os'] == 'ironware' || $device['os_type'] == 'ironware') {
+if ($device['os'] == 'ironware' || $device['os_type'] == 'ironware' || $device['os'] == 'nos') {
```

Comment:
Seeing this now makes me realize the last part of the IF-statement stands in conflict with nos.inc.php that's also does discovery for nos.

Consider a Brocade VDX device, will it now be discovered as ironware or nos?